spec/rubocop/cop/rails/implemented_migration_spec.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::ImplementedMigration, :config do
+  it 'does not register an offense with a change method' do
+    expect_no_offenses(<<~RUBY)
+      class SomeMigration < ActiveRecord::Migration[6.0]
+        def change
+          add_column :users, :email, :text, null: false
+        end
+      end
+    RUBY
+  end
+
+  it 'registers an offense with only an up method' do
+    expect_offense(<<~RUBY)
+      class SomeMigration < ActiveRecord::Migration[6.0]
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Migrations must contain either a change method, or both an up and a down method.
+
+        def up
```

Comment:
Can you remove the redundant blank line in this PR' test examples?

```suggestion
      class SomeMigration < ActiveRecord::Migration[6.0]
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Migrations must contain either a change method, or both an up and a down method.
        def up
```